tests/jest/background/vpncontroller/vpncontroller.test.mjs

```diff
@@ -191,25 +192,26 @@ describe("handleBridgeResponse", () => {
   });
 
   test("handles vpn-client-down when client is alive", async () => {
-    const state = { alive: true, installed: true };
-
+    const state = property({ alive: true, installed: true });
     await vpnController.handleBridgeResponse(
       { status: "vpn-client-down" },
       state
     );
 
-    expect(vpnController.state.value.installed).toBe(true);
-    expect(vpnController.state.value.alive).toBe(false);
+    expect(state.value.installed).toBe(true);
+    expect(state.value.alive).toBe(false);
   });
```

Comment:
Now how did the unit test not catch that, welll : 
```
expect(new VpnController().state.value.alive).toBe(false);
expect(new VpnController().state.value.alive).toBe(false);
```
I should have made this function static from the beginning >_<